src/DynamoCoreWpf/Views/SplashScreen/SplashScreen.xaml.cs

```diff
@@ -392,6 +400,17 @@ internal async void SetSignInStatus(bool status)
                 $"signInStatus: \"" + status + "\"})");
             }
         }
+        /// <summary>
+        /// Enable or not the SignIn button on the fly.
+        /// </summary>
+        /// <param name="enabled"></param>
+        internal async void SetSignInEnable(bool enabled)
+        {
+            if (webView?.CoreWebView2 != null)
+            {
+                await webView.CoreWebView2.ExecuteScriptAsync("window.setEnableSignInButton({" + $"enable: \"" + enabled + "\"})");
```

Comment:
can't you use a string literal here to avoid the escaping weirdness? `@`

Reply:
> can't you use a string literal here to avoid the escaping weirdness? `@`

just formatted as the JS object requires, like the similar calls.